.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: "publish"
+
+on:
+  push:
```

Comment:
I think this will try to publish on every commit to master/release branch, or does it just publish when you change the version?

Reply:
I think you have right, and publish runner is called a bit too often.. a better approach could push when a new tag release is generate.. what do you think? I'll take a look on gitlab runner yml setup

Reply:
For the BDK project we just have a manual process in the https://github.com/bitcoindevkit/bdk/blob/master/DEVELOPMENT_CYCLE.md document.  I like the idea of triggering based on tag creation also but unless you're releasing often may still be overkill.

Reply:
Removed, so we are ready for tagging

Reply:
I still see the `on: push:` did you push you changes ?

Reply:
sorry to be late, commit dropped